src/main/java/org/broadinstitute/hellbender/tools/spark/sv/evidence/BreakpointEvidence.java

```diff
@@ -877,41 +890,32 @@ public SameStrandPair read( final Kryo kryo, final Input input, final Class<Same
 
     @DefaultSerializer(WeirdTemplateSize.Serializer.class)
     public static final class WeirdTemplateSize extends DiscordantReadPairEvidence {
-        private final int templateSize;
         private final int mateStartPosition;
         private final boolean mateReverseStrand;
         private static final int WEIRD_TEMPLATE_SIZE_WEIGHT = ReadEvidence.SINGLE_READ_WEIGHT;
 
         WeirdTemplateSize( final GATKRead read, final ReadMetadata metadata ) {
             super(read, metadata, WEIRD_TEMPLATE_SIZE_WEIGHT);
-            this.templateSize = read.getFragmentLength();
             this.mateStartPosition = read.getMateStart();
             this.mateReverseStrand = read.mateIsReverseStrand();
         }
 
         private WeirdTemplateSize( final Kryo kryo, final Input input ) {
             super(kryo, input);
-            templateSize = input.readInt();
             mateStartPosition = input.readInt();
             mateReverseStrand = input.readBoolean();
         }
 
         @Override
         protected void serialize( final Kryo kryo, final Output output ) {
             super.serialize(kryo, output);
-            output.writeInt(templateSize);
             output.writeInt(mateStartPosition);
             output.writeBoolean(mateReverseStrand);
         }
 
         @Override
         public String toString() {
-            return super.toString() + "\tTemplateSize\t" + target + "\t" + templateSize;
-        }
-
-        @Override
-        public String stringRep(final ReadMetadata readMetadata, final int minEvidenceMapq) {
-            return super.stringRep(readMetadata, minEvidenceMapq) + "\t" + templateSize;
+            return super.toString() + "\tTemplateSize\t" + getTemplateSize();
```

Comment:
Why not use `super.stringRep` which already includes the template size?

Reply:
The stringRep() functions were added so that I could dump additional info without messing with the toString() functions which you guys have been using for debug purposes. I've been trying to ensure that the two functions are largely independent and thought that it was better to not have them call each other.

Also: I just noticed that I accidentally deleted the print-out of "target", which I've reverted.